src/app/core/abstractions/entity-parser.service.ts

```diff
@@ -5,7 +5,7 @@ import { Part } from '../models/part'
 import { Piece } from '../models/piece'
 import { AdLibPiece } from '../models/ad-lib-piece'
 
-export abstract class EntityParser {
+export abstract class EntityParserService {
```

Comment:
I though this was the kind of naming you didn't want? Like you don't want `xHandlerService` etc?

Reply:
Correct :D I must have accidentally have accepted the rename prompt in WebStorm :grin:
I have changed it back.